src/components/background-control/editor.scss

```
@@ -271,58 +316,42 @@ body .maxi-background-control {
 			background: var(--maxi-mid-green);
 		}
 	}
 }
+
+@import '../focal-point-picker';
+
+// FocalPointPicker styling for background image layer
+.maxi-background-control__image-layer {
+	.maxi-focal-point-picker {
+		@include stacking-context;
+		margin-bottom: 8px;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)